getting-started/setup-multi-node-basic.md

```diff
@@ -0,0 +1,111 @@
+# Set up multi-node TimescaleDB [](basic-multi-node-setup)
+
+A multi-node TimescaleDB implementation consists of: 
+- One access node to handle ingest, data routing and act as an entry 
+point for user access; 
+- One or more data nodes to store and organize distributed data.     
+
+All nodes begin as single-node TimescaleDBs, i.e., hosts with a running 
+PostgreSQL instance and a loaded TimescaleDB extension.  More detail on 
+the architecture can be found [here][architecture]. For the below setup 
+instructions, "data node" and "access node" will be used to refer to both the 
+server running TimescaleDB, as well as the TimescaleDB instance itself.
+The steps for creating a basic multi-node setup are as follows:
+
+## Basic setup
+1. Configure data nodes for communication with the access node
+1. Initialize data nodes from the access node 
+
+### Prerequisites
+- One PostgreSQL instance to act as an access node
+- One or more PostgreSQL instances to act as data nodes
+- TimescaleDB [installed][install] and [set up][setup] on all nodes
+- Access to a superuser role (e.g. `postgres`) on all nodes
+
+### 1. Configure data nodes for node-to-node communication
+To enable communication between the access node and the data node(s), the
+data node must be configured to authorize connections from the access node.  The 
+simplest way is to use "trust" authentication which provides unencumbered client 
+access to the nodes. This method must be applied to every data node.
+
+>:WARNING: The "trust" authentication method allows insecure access to all 
+nodes.  For production implementations, please use more secure 
+methods of authentication (see [advanced setup][] for examples).
+
+#### Edit authentication configuration file on data nodes
+Client authentication is usually configured in the `pg_hba.conf`([reference doc]
+[postgresql-hba]) file located in the data directory.  If the file is not located 
+there, access the instance (psql) and execute the command:
+
+```sql
+SHOW hba_file;
+``` 
+
+To enable "trust" authentication, add a line to `pg_hba.conf` to allow the
+access to the instance:
+
+```
+# TYPE  DATABASE  USER  ADDRESS         METHOD
+host    all       all   {access_node_ip}   trust
```

Comment:
Suggest to use a TEST-NET address instead of a replaceable, since you need to repeat it below anyway.
```suggestion
# TYPE  DATABASE  USER  ADDRESS         METHOD
host    all       all   192.0.2.20   trust
```